src/main/java/moze_intel/projecte/PECore.java

```diff
@@ -48,12 +48,7 @@
 import moze_intel.projecte.network.PacketHandler;
 import moze_intel.projecte.network.ThreadCheckUUID;
 import moze_intel.projecte.network.ThreadCheckUpdate;
-import moze_intel.projecte.network.commands.ClearKnowledgeCMD;
-import moze_intel.projecte.network.commands.DumpMissingEmc;
-import moze_intel.projecte.network.commands.RemoveEmcCMD;
-import moze_intel.projecte.network.commands.ResetEmcCMD;
-import moze_intel.projecte.network.commands.SetEmcCMD;
-import moze_intel.projecte.network.commands.ShowBagCMD;
+import moze_intel.projecte.network.commands.*;
```

Comment:
For the most part in this project we use fully qualified imports rather than wildcards.